src/components/transactions/TxDetails/TxNote.tsx

```diff
@@ -0,0 +1,44 @@
+import { DataRow } from '@/components/common/Table/DataRow'
+import useAsync from '@/hooks/useAsync'
+import { useCurrentChain } from '@/hooks/useChains'
+import { isMultisigDetailedExecutionInfo } from '@/utils/transaction-guards'
+import { Box, Divider } from '@mui/material'
+import type { TransactionDetails } from '@safe-global/safe-gateway-typescript-sdk'
+
+const TxNote = ({ txDetails }: { txDetails: TransactionDetails }) => {
+  const currentChain = useCurrentChain()
+  const txService = currentChain?.transactionService
+
+  let safeTxHash = ''
+  if (isMultisigDetailedExecutionInfo(txDetails.detailedExecutionInfo)) {
+    safeTxHash = txDetails.detailedExecutionInfo?.safeTxHash
+  }
+
+  const [data] = useAsync(() => {
+    if (!safeTxHash || !txService) return
+    return fetch(`${txService}/api/v1/multisig-transactions/${safeTxHash}`).then((res) => res.json())
```

Comment:
I'm using the tx-service directly here because CGW doesn't mirror the `origin` field. It can be easily added there.